cluster/os-3.9.0/provider.sh

```diff
@@ -2,7 +2,11 @@
 
 set -e
 
-image="os-3.9.0@sha256:507263b67ddad581b086418d443c4fd1b2a30954e8fddff12254e0061a529410"
+image="os-3.9.0@sha256:d55fc1bef8a9ab327c5f213deec75ad9cc3a1c258593b9fd966b11bef6010bd6"
+if [[ "${CRIO}" == "true" ]]; then
+    image="os-3.9.0-crio@sha256:edea5ea811eaa150f0ae56022188d2e33872d855b84e2a8ffb5294f868b23840"
```

Comment:
if crio support involves a different image entirely, shouldn't we just consider os-3.9.0-crio a separate provider instead of a ENV VAR that toggles behavior on/off?

Reply:
I thought about it, but I just did not want to create additional empty directories with that will include only `image` variable. I can update `os-3.9.0` readme file with relevant information about CRI-O deployment.
If you insist on an additional provider, I can create it as well.

Reply:
If we don't use a separate provider, then we'd have to support this CRIO option across all providers, even if that means just creating a check to fail in situations where CRIO isn't supported. 

a separate provider avoids all that. I don't have any problem with a separate provider that's essentially just an image variable.

Reply:
Created separate provider for os with cri-o environment.